### ISSUE TITLE
Question on improvement for specific image

### ISSUE THREAD

**Issue by partizanos** [sadeepj/crfasrnn_keras#37]
Hello I am working on a master project and I am interested in applying the solution into static pictures of a plant.
The pretrained model has already very good results but I am wondering what are some possible ways to improve it. I am wondering if there are some kind of techniques to improve it further. A second issue comes with the different lighting during the day that changes the plant colors.
Thank you in advance for your time. 

![image](https://user-images.githubusercontent.com/5407188/41127675-fb05f13a-6aab-11e8-95a2-61a6a6c94e76.jpg)
![labels](https://user-images.githubusercontent.com/5407188/41127679-fd64b222-6aab-11e8-80a0-112ee51dfb0b.png)

**Comment by sadeepj** [comment]
@partizanos My suggestion would be to fine-tune/retrain the network with some training data from your own domain. The publicly available model was trained with COCO images and Pascal VOC images, which might look quite different to the images you're testing on.
